code/modules/surgery/experimental_dissection.dm

```diff
@@ -78,6 +78,21 @@
 	target.apply_damage(80, BRUTE, L)
```

Comment:
I kind of hate that all of this code is ran in the surgery itself, seems too magical to have your ID automagically paid. There should be a Claim Credits button on the surgery console that's created when the surgery is finished, and can be swiped by anybody.